docs/guide/index.asciidoc

```diff
@@ -39,30 +35,24 @@ image::apm-architecture.png[Architecture of Elastic APM]
 
 *APM agents* are open source libraries written in the same language as your service.
 You install them into your service as you would install any other library.
-The agents hook into your service and start collecting performance metrics and errors once it starts.
-All data collected by agents is buffered for a short period and sent on to APM Server.
-This happens once per minute,
-by default.
+They instrument your code and collect performance data and errors at runtime.
+This data is buffered for a short period and sent on to APM Server.
 
 Out of the box,
-APM automatically instruments most popular web frameworks,
+APM automatically instruments web frameworks,
 database drivers,
 calls to caching servers,
 and HTTP libraries for requests to external services.
 Additionally,
-agents provide an API that you can use to manually instrument anything else you find interesting.
+agents provide an API to manually instrument anything else you find interesting.
 
 *APM Server* is an open source application written in Go which typically runs on dedicated servers.
-It listens on port 8200 by default and receives data from agents periodically through a JSON API.
-APM Server creates documents from the data received from agents and stores them in an Elasticsearch cluster.
+It receives data from the agents through a JSON API, process it, and stores it in Elasticsearch.
```

Comment:
You're introducing a couple of unclear antecedents in this sentence. I think the original version is easier to parse (though I agree it's wordy). I'd keep this as a couple of sentences, but tighten up the wording a bit and remove extraneous info.